### PR TITLE
docs: use extension "tab" to display example codes in "Schema"

### PR DESCRIPTION
**Pull request by graczhual**

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-2gh5pg7 调研 tab 方式展示部分场景下的样例代码](https://app.clickup.com/t/2gh5pg7)

File: docs/source/features/sheet_management/schema.rst [hunk 1]
```diff
@@ -378,51 +436,10 @@ Please see :ref:`features/sheet_management/dataframe:File Operation` for more de
  Schema Methods
 ****************
 
-to/from Python Object
-=====================
-PortexType provides methods to convert to or init from python object:
+Convert to/from Other Types
```
Comment (Lee-000) [inline docs/source/features/sheet_management/schema.rst:439]: What about just `convert`? `Other Types` may not be very accurate.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -107,7 +107,7 @@ repos:
         exclude: "(^docs/source/_templates/)"
         args:
           [
-            "--ignore-directives=automodule",
```
Comment (linjiX) [inline .pre-commit-config.yaml:110]: the ignore of automodule is removed? Does it means we will never use automodule in this repo?

Comment (graczhual) [reply .pre-commit-config.yaml:110]: because wu use autoapi to replace "autoclass", "automobile"

**Review by linjiX — APPROVED** [review]

**Review by Lee-000 — APPROVED** [review]

**Pull request merged**